lib/modules/platform/github/scm.ts

```diff
@@ -1,8 +1,13 @@
+import is from '@sindresorhus/is';
 import * as git from '../../../util/git';
 import type { CommitFilesConfig, LongCommitSha } from '../../../util/git/types';
 import { DefaultGitScm } from '../default-scm';
 import { commitFiles, isGHApp } from './';
 
+export function sanitizeMentions(input: string): string {
+  return input.replaceAll('@', '@\u{8203}');
```

Comment:
How does this look if you do e.g. "git log"? Is it invisible?

Reply:
Everything looks correct:

<img width="666" alt="image" src="https://github.com/user-attachments/assets/312d754f-06dc-48f6-a9b3-1be97f004b2c">

<img width="277" alt="image" src="https://github.com/user-attachments/assets/967a95b5-1457-473e-b39d-b9bc17d67b11">

Reply:
On some terminals, the space is rendered, though

Reply:
Maybe search for `/\b@/` (or even `/\b@\S/`)? Asking as a user of "non-web terminal"

Reply:
The problem isn't with search, the problem is with display. I'm pretty sure it can't be rendered equally nice in both the web and terminal (which could be browser-based or the classic GUI ones). After all, zero-width space and backspace characters are just hacks. Having space characters rendered in terminal is the less evil compared to pinging lots of people mentioned in the changelog.

Reply:
I mean you could likely reduce the impact - because not every `@` character starts a user mention (although my regex suggestions were wrong)

Reply:
Okay, for now we reverted the change and won't iterate on the underlying problem as far as it's a singular event of some user configuring Renovate in a pretty non-conventional way